packages/flutter/lib/src/material/tabs.dart

```diff
@@ -1100,7 +1108,15 @@ class _TabBarState extends State<TabBar> {
       );
     }
 
-    return tabBar;
+    // Wrap in Material if the tabBar has an elevation
+    if (widget.elevation > 0) {
```

Comment:
This needs a test.  It also is now failing analysis and needs to be brought up to date.  Can you update it?